pkg/apiserver/infrastructure/clients/kubeclient_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clients
+
+import (
+	"context"
+	"sync"
+
+	"github.com/oam-dev/kubevela/pkg/apiserver/config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Test kube client", func() {
+
+	PIt("Test client-side throttling", func() {
```

Comment:
why pending this test

Reply:
This test case depends on the cluster gateway, the default environment of the unit test without the cluster gateway